src/StructuredLogger/BinaryLogger/BuildEventArgsReader.cs

```diff
@@ -1245,6 +1248,49 @@ private BuildEventArgs ReadEnvironmentVariableReadEventArgs()
             return e;
         }
 
+        private BuildEventArgs ReadBuildCheckTracingEventArgs()
+        {
+            var fields = ReadBuildEventArgsFields(readImportance: true);
+            var rawTracingData = ReadStringDictionary() ?? new Dictionary<string, string>();
+
+            var e = new BuildCheckTracingEventArgs(rawTracingData.ToDictionary(

```

Comment:
Is this class already known to viewer?

Reply:
I have added it as a part of this PR
https://github.com/KirillOsenkov/MSBuildStructuredLog/pull/797/files#diff-3280c719e5b13a87456db15d9462ea10354769ef6fc8d731c436ac74a183bfa7:~:text=internal%20sealed%20class-,BuildCheckTracingEventArgs,-(Dictionary%3C

Reply:
![image](https://github.com/user-attachments/assets/e9716999-6757-4398-a35b-7959b80dc7f1)